src/demos/AirshipModal.tsx

```diff
@@ -103,15 +105,16 @@ export class AirshipModal extends React.Component<Props, State> {
           function upValue(keyboardHeight: number): number {
             return Math.max(keyboardHeight, downValue)
           }
-
+          const styleOverride = this.props.styleOverride || {}
```

Comment:
Destructuring assignment is nicer, like `const { styleOverride = {} } = this.props`.